src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
Ensure that the 'table' element exists in the DOM before executing this script to avoid potential null reference errors.